packages/react-styled-ui/src/TMIcon/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Icon from '../Icon';
+
+const TMIcon = React.forwardRef((
+  {
+    icon,
+    size,
+    ...props
+  },
+  ref
+) => {
+  if (size !== undefined) {
+    if (typeof size === 'number') {
+      size = `${size || 0}px`;
+    }
+  }
+  return (
+    <Icon
+      ref={ref}
+      icon={`_core.tmicon-${icon}`}
```

Comment:
@cheton Let me know if we should remove the `_core` prefix here. If yes, I will make new changes accordingly.

Reply:
yes! please remove it if possible

Reply:
Will do.